skimage/exposure/exposure.py

```diff
@@ -647,6 +647,10 @@ def is_low_contrast(image, fraction_threshold=0.05, lower_percentile=1,
         if image.shape[2] == 3:
             image = rgb2gray(image)
 
+    if image.dtype == bool:
+        return image.any() and (image == False).any()
+
```

Comment:
Maybe I'm missing something, but if a boolean image has both True and False values, then the function should return False, shouldn't it? I have the impression that the returned value should be
```suggestion
return not(image.any() and (image == False).any())
```

Reply:
anyway a test (as @jni suggested) could either prove me that I'm wrong :-) or check that the behaviour is as expected

Reply:
When `image.dtype == bool`, `is_low_contrast` is `True` when the image is constant: all filled with `True` or with `False`, _ie_ when `False` and `True` are not both in `image`.

So I think that you are right @emmanuelle!

BTW, I think that this test must take place at the beginning of the function, even before checking for image dimensions, because `rgb2gray` will convert the image to floating point...